autosubmit/job/job.py

```diff
@@ -1673,11 +1673,15 @@ def process_scheduler_parameters(self, job_platform, chunk):
             # Get the max threads, each element can be a str or int
             self.het['NUMTHREADS'] = list()
             if len(self.threads) == 1:
-                for x in range(self.het['HETSIZE']):
-                    self.het['NUMTHREADS'].append(self.threads)
+                if(self.threads > 1):
```

Comment:
No need for parentheses in Python 👍 

And isn't what you are doing invalid for a list?

The code above has a `if type(self.threads) is list`, so we know that `self.threads` is a list. Above we check the length, but here you compare the list to `> 1`?